### PR TITLE
Remove dev flag to show the new UI changes

### LINKED ISSUE #1276: Fix: remove dev flag from mine page to show the new UI changes
### Issue Description

The [mine](https://status.realdevsquad.com/mine) page has all the necessary changes by this PR #1264. The changes are stable in production but It is still under feature flag.

### Expected Behavior

When you go to mine page, It should show this UI without dev flag.
Screenshot 2024-10-21 at 6 54 10 PM

### Current Behavior
Right now, It shows this UI.
Screenshot 2024-10-21 at 6 57 28 PM

### Reproducibility

 - [x] This issue is reproducible
 - [ ] This issue is not reproducible

### Steps to Reproduce
1. Go to [mine](https://status.realdevsquad.com/mine) page
2. add dev?=true in url query to see the new changes

### Severity/Priority
 - [ ] Critical
 - [ ] High
 - [ ] Medium
 - [x] Low

### Additional Information

The code for old UI can be removed.
### Checklist

- [x] I have read and followed the project's code of conduct.
- [x] I have searched for similar issues before creating this one.
- [x] I have provided all the necessary information to understand and reproduce the issue.
- [x] I am willing to contribute to the resolution of this issue.

### PR DESCRIPTION
**Pull request by pankajjs**
Date: 21 Oct, 2024

Developer Name: @pankajjs 

---

## Issue Ticket Number

 - Closes #1276 
## Description
This PR removes dev flag to from mine page to show the new UI changes.

### Documentation Updated?

- [ ] Yes
- [x] No

### Under Feature Flag

- [ ] Yes
- [x] No

### Database Changes

- [ ] Yes
- [x] No

### Breaking Changes

- [ ] Yes
- [x] No

### Development Tested?

- [x] Yes
- [ ] No

## Screenshots
<details>
<summary>Working Video</summary>

https://github.com/user-attachments/assets/21aaa51b-3715-4de9-b805-6d7808b8cda5

</details>

## Test Coverage
<details>
<summary>Screenshots</summary>
Screenshot 2024-10-21 at 8 21 39 PM

Screenshot 2024-10-21 at 8 21 29 PM

</details>

### CODE REVIEW

File: src/pages/mine/index.tsx [hunk 1]
```diff
@@ -91,39 +89,8 @@ const ContentDev = () => {
...
-                <Card
-                    content={item}
-                    key={item.id}
-                    shouldEdit={false}
-                    onContentChange={undefined}
-                />
-            ))}
-        </>
-    );
-}
```
Comment (prakashchoudhary07) [inline src/pages/mine/index.tsx:107]: Are we sure, that this is not being used any where?

Comment (pankajjs) [reply src/pages/mine/index.tsx:107]: Yes @prakashchoudhary07, I have double checked. It is not being used anywhere.

File: src/pages/mine/index.tsx [hunk 2]
```diff
@@ -132,11 +99,7 @@ const Mine: FC = () => {
                 {isAuthenticating ? (
                     <Loader />
                 ) : isLoggedIn ? (
-                    dev ? (
-                        <ContentDev />
-                    ) : (
-                        <Content />
-                    )
+                    <ContentDev />
```
[the remark this answers is not in the document]
Comment (pankajjs) [reply src/pages/mine/index.tsx:102]: In the current changes, I have -
 - removed the unused component
 - removed the dev flag condition and showing the new component
 - refactored the tests for new UI component
 - removed the tests for unused component
 @prakashchoudhary07

[the remark this answers is not in the document]
Comment (samarpan1738) [reply src/pages/mine/index.tsx:102]: @pankajjs Can we rename `ContentDev` component to `Content` since we don't have the dev flag anymore?

**Review by prakashchoudhary07 — APPROVED** [review]

**Pull request merged**